From e844dbc5c896b1bb61ca8c10d153ca3c5c6df86c Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Wed, 20 Dec 2017 15:04:39 +0900
Subject: [PATCH 604/909] arm64: dts: r8a7795-salvator-x: Add MMNGRBUF driver
 node

commit 7baf223e3720ee46fe9599f8afebfcb705e0d452 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This patch adds MMNGRBUF external kernel module for the R8A7795
ES2.0/3.0 SoC on the Salvator-X board.

Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Hai Nguyen Pham <hai.pham.ud@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a7795-salvator-x.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a7795-salvator-x.dts b/arch/arm64/boot/dts/renesas/r8a7795-salvator-x.dts
index b10cb9a698be..6f1b7fe55dca 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795-salvator-x.dts
+++ b/arch/arm64/boot/dts/renesas/r8a7795-salvator-x.dts
@@ -72,6 +72,10 @@
 		compatible = "renesas,mmngr";
 		memory-region = <&mmp_reserved>, <&lossy_decompress>;
 	};
+
+	mmngrbuf {
+		compatible = "renesas,mmngrbuf";
+	};
 };
 
 &du {
-- 
2.17.1

